### ISSUE TITLE
[Feature Unlock] Sequoia - Continuity camera blurs and reconnects every 12 secs on WiFi

### ISSUE THREAD

**Issue by ruler666** [acidanthera/bugtracker#2448]
Hi,

It seems that there are many of us (if not all) with Sequoia installs whom have this problem (source - insanelymac.com forum). iPhone camera is successfully recognized as a source camera, it can be selected in any tool all over the OS, starts displaying image, and after 12 secs we see a blurry picture on the Hack screen, with message reconnecting on iPhone and it starts working again for another 12 secs. For most, it works flawlessly with USB connection.

Someone somewhere on the forum mentioned that devs might not even be aware of the problem, so this is my attempt of making the key people aware. 

I can attach any logs (just say what you need) and|or video of the problem, if necessary. 

Thanks!

**Comment by daydog59** [comment]
yes I also have the 12 sec blurry than clear than 12 sec blurry on Sequoia 15.1.1

**Comment by esafeddie** [comment]
My own experience with OCLP is flawless within Sonoma both wireless and USB connection.
In Sequoia USB connection works as should without any problems, but wireless connection refresh every 10 to 12 seconds with a blurry image at times as reported by other users. This behavior has been present since the adaptation for Sequoia.

**Comment by vit9696** [comment]
Probably out of the scope of this project. My best guess would be some bug in Wireless drivers (BT or Wi-Fi)?
